core/drivers/crypto/crypto_api/include/drvcrypt.h

```diff
@@ -55,7 +55,8 @@ enum drvcrypt_algo_id {
 	CRYPTO_MATH,	 /* Mathematical driver */
 	CRYPTO_CIPHER,   /* Cipher driver */
 	CRYPTO_ECC,      /* Asymmetric ECC driver */
-	CRYPTO_MAX_ALGO  /* Maximum number of algo supported */
+	CRYPTO_DH,       /* Asymmetric DH driver */
+	CRYPTO_MAX_ALGO	 /* Maximum number of algo supported */
```

Comment:
did something changed in this line?

Reply:
It must a space instead of a tab or something like that .. I will check